src/util/bucket_logs_utils.js

```diff
@@ -50,17 +50,16 @@ async function export_logs_to_target(fs_context, s3_connection, bucket_to_owner_
  * This function gets a persistent log file, will go over it's entries one by one,
  * and will upload the entry to the target_bucket using the provided s3 connection
  * in order to know which user to use to upload to each bucket we will need to provide bucket_to_owner_keys_func
- * @param {import('../sdk/config_fs').ConfigFS} config_fs
+ * @param {nb.NativeFSContext} fs_context config_fs
  * @param {AWS.S3} s3_connection
  * @param {string} log_file
  * @param {function} bucket_to_owner_keys_func
  * @returns {Promise<Boolean>}
  */
-async function _upload_to_targets(config_fs, s3_connection, log_file, bucket_to_owner_keys_func) {
+async function _upload_to_targets(fs_context, s3_connection, log_file, bucket_to_owner_keys_func) {
     const bucket_streams = {};
     const promises = [];
     try {
-        const fs_context = config_fs.fs_context;
         const file = new LogFile(fs_context, log_file);
```

Comment:
why not just pass config_fs.fs_context? will you need to do other configuration file usage in the future?

Reply:
file is shared between containerized and none - as we don't have config_fs there. I need to use only fs_context.